### PR TITLE
Remove `manual_grasp_link`

### PR DESCRIPTION
**Pull request by sea-bass**
Requires https://github.com/PickNikRobotics/picknik_accessories/pull/4

### CODE REVIEW

File: src/picknik_ur_base_config/description/picknik_ur.xacro [hunk 1]
```diff
@@ -98,14 +98,6 @@
     <origin xyz="0.0 0.0 0.134" rpy="0.0 0.0 ${pi}" />
   </joint>
 
-  <!-- Moveit Studio requires a link name called manual_grasp_link to perform quick tasks such as inspect surface -->
```
Comment (MarqRazz) [inline src/picknik_ur_base_config/description/picknik_ur.xacro:101]: Should we preserve this comment for `grasp_link` in all configs?

Comment (sea-bass) [reply src/picknik_ur_base_config/description/picknik_ur.xacro:101]: The comment is still in the `gripper_and_camera.xacro`, `robotiq_2f_85_gripper.urdf`, and `config.yaml`, so I'm OK with removing this whole block.

To me, it doesn't make much sense to have a standalone comment where there no longer is a `manual_grasp_link` or `grasp_link` definition.

What do you think?

Comment (MarqRazz) [reply src/picknik_ur_base_config/description/picknik_ur.xacro:101]: Grasp link is defined on line 94 and this URDF is calling
`<xacro:include filename="$(find robotiq_description)/urdf/robotiq_2f_85_macro.urdf.xacro" />` not the one in picknik_accessories. FYI if you want to use PickNik's xacro we need the ability to pass in the `comm_port` value.

Comment (sea-bass) [reply src/picknik_ur_base_config/description/picknik_ur.xacro:101]: Gotcha! Added the comment to line 94.

**Review by MarqRazz — COMMENTED** [review]
Tested several objectives on the `picknik_ur_gazebo_config` and they all work except for `Open Cabinet Door`. It fails with a BT error
```
Error: Failed to create behavior tree for Objective `Open Cabinet Door`. Reason: BehaviorTree error while creating tree: Possible typo? In the XML, you tried to remap port "screw_origin_pose" in node [GetMoveAlongArcSubframes / GetMoveAlongArcSubframes], but the manifest of this node does not contain a port with this name.
```

Is this something we want to fix here?

File: src/picknik_ur_base_config/description/picknik_ur.xacro (hunk 1 above)
Comment (MarqRazz) [reply src/picknik_ur_base_config/description/picknik_ur.xacro:101]: Thanks for making the change there, but none of the robots here use that xacro which seems a little weird.

**Comment by sea-bass** [comment]
I think this is failure is unrelated, and due to the fact that the ports in `GetMoveAlongArcSubframes` have been renamed without having updated this Objective in the Gazebo config.

```yaml
            <input_port name="target_grasp_pose" default="{target_grasp_pose}">Pose representing the grasp point.</input_port>
            <input_port name="hinge_axis_pose_start" default="{hinge_axis_pose_start}">Pose marking the start of the hinge axis.</input_port>
            <input_port name="hinge_axis_pose_end" default="{hinge_axis_pose_end}">Pose marking the end of the hinge axis.</input_port>
            <output_port name="move_along_arc_subframes" default="{move_along_arc_subframes}">Subframes representing the ordered screw (hinge) origin and axis, along with a grasp subframe.</output_port>
```

@chancecardona Just a heads-up, in case this is something not on your radar with your upcoming refactors (but I'm sure you've handled this).

**Review by MarqRazz — APPROVED** [review]

**Comment by chancecardona** [comment]
> I think this is failure is unrelated, and due to the fact that the ports in GetMoveAlongArcSubframes have been renamed without having updated this Objective in the Gazebo config.

https://github.com/PickNikRobotics/moveit_studio_ur_ws/blob/184821d982e6b80bb826f3635c5c6d6fcc5cb46a/src/picknik_ur_site_config/objectives/open_cabinet_door.xml I have updated the open_cabinet_door objective now to work with the changes like you mentioned (at least until it fails to plan in sim!). So this should be taken care of by the gibson removal PR's! :+1:

**Pull request merged**